### ISSUE TITLE
Method displayUsingLabels for select fields doesn't work

### ISSUE THREAD

**Issue by David-YesWeDev** [optimistdigital/nova-translatable#15]
Hi !

I'm having a problem with select fields, when i set the displayUsingLabels method, it's doesn't work.
On the detail screen, i still have the value displayed.

Here is the field i set up :

```
Select::make('Présence sur page d\'accueil', 'is_on_home')
->options([
  0 => 'Non',
  1 => 'Oui',
])
->displayUsingLabels()
->translatable()
->hideFromIndex(),
```

And here is how the data is displayed in the detail page : 

![select-label-bug](https://user-images.githubusercontent.com/32599362/78798378-3434af00-79b9-11ea-8a93-0ef4693173bb.png)

Thanks for the help !

**Comment by Tarpsvo** [comment]
Hi! That's really weird. I copied your code and it worked perfectly for me.

![image](https://user-images.githubusercontent.com/2018660/79225039-cd097580-7e64-11ea-8f9d-69af9106661a.png)

Can you confirm you still have this issue?

**Comment by David-YesWeDev** [comment]
> Hi! That's really weird. I copied your code and it worked perfectly for me.
> 
> ![image](https://user-images.githubusercontent.com/2018660/79225039-cd097580-7e64-11ea-8f9d-69af9106661a.png)
> 
> Can you confirm you still have this issue?

Hi ! Yes the issue still exist but you can view it only in the resource **detail** page. Can you reproduce it ?

**Comment by Tarpsvo** [comment]
Hi! Released version 1.5.0 with the fix. Good luck!

**Comment by David-YesWeDev** [comment]
Hi ! Thanks for the fix !
